spec/requests/onboardings_spec.rb

```diff
@@ -23,17 +23,5 @@
       expect(response.body).to include("data-community-background")
       expect(response.body).to include("data-community-name")
     end
-
-    it "contains proper data attribute values if the onboarding config is present" do
-      allow(SiteConfig).to receive(:onboarding_logo_image).and_return("onboarding_logo_image.png")
-      allow(SiteConfig).to receive(:onboarding_background_image).and_return("onboarding_background_image.png")
-
-      sign_in user
-      get onboarding_url
-
-      expect(response.body).to include(SiteConfig.community_description)
-      expect(response.body).to include(safe_logo_url(SiteConfig.onboarding_logo_image))
```

Comment:
**question (non-blocking):** should we update this spec to check for `SiteConfig.secondary_logo_url` instead of deleting it?

Reply:
Oh yeah, good call. I'll update this.

Reply:
Agree with @atsmith813, we should check that `secondary_logo_url` does in fact render in the onboarding request body 👍🏽

Reply:
Fixed this.